src/components/Styles/Books.css

```diff
@@ -1,8 +1,8 @@
 .book-container {
   width: 90%;
-  border: 1px solid #888;
+  border: 1px solid lightcyan;
   padding: 2%;
-  margin: 30px auto;
+  margin: 20px auto;
   box-shadow: -1px -1px 2px 1px  #888;
   display: flex;
   justify-content: space-between;
```

Comment:
- [ ] Good job implementing the styling for your application, however as mentioned in the project requirements _**"Your final design should match Zeplin's design (this includes all elements + the same styling)"**_
please the current design I have tried to highlight some of the sections that should match the design :point_down: 

![WhatsApp Image 2023-01-06 at 00 03 41](https://user-images.githubusercontent.com/97521557/210856436-2cb9b1d8-47d9-4beb-8493-7ea40022e810.jpeg)

Please refer to this design and try to follow the styling as close to it as possible, you can inspect to get the CSS styles applied here is [the link to the design ](https://pedantic-beaver-cb923b.netlify.app/#/) here is a picture :point_down: 
![image](https://user-images.githubusercontent.com/97521557/210856798-aa9a5861-a824-451c-b024-0bc32c3e88b5.png)